src/Logic/Settings.cs

```diff
@@ -1316,9 +1316,11 @@ private static Settings CustomDeserialize(string fileName)
             subNode = node.SelectSingleNode("SubtitleMaximumDisplayMilliseconds");
             if (subNode != null)
                 settings.General.SubtitleMaximumDisplayMilliseconds = Convert.ToInt32(subNode.InnerText);
-            subNode = node.SelectSingleNode("MininumMillisecondsBetweenLines");
+            subNode = node.SelectSingleNode("MinimumMillisecondsBetweenLines");
+            if (subNode == null) // TODO: Remove in 3.5
+                subNode = node.SelectSingleNode("MininumMillisecondsBetweenLines");

```

Comment:
???
I'm afraid, I don't understand what you mean.
These two 'compatibility lines' are there to be able to use the value from the existing Settings.xml file (with the tag named Mini**n**umMillisecondsBetweenLines).
When the Settings.xml file is re-generated, the new tag (Mini**m**umMillisecondsBetweenLines) will be used.